iothub/service/src/Messaging/TransportType.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Transport types supported by MessagingClient.
+    /// </summary>
+    /// <remarks>
+    /// Amqp and Amqp over WebSocket.
+    /// </remarks>
+    public enum TransportType
+    {
+        /// <summary>
+        /// Advanced Message Queuing Protocol transport.
```

Comment:
Call out what port this uses in the remarks. Users with firewalls may have this port blocked.